modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/repositories/appcatalog/ApplicationInterfaceRepository.java

```diff
@@ -122,15 +130,15 @@ public ApplicationModule getApplicationModule(String moduleId) throws AppCatalog
         ApplicationModuleRepository applicationModuleRepository = new ApplicationModuleRepository();
         Map<String, Object> queryParameters = new HashMap<>();
         queryParameters.put(DBConstants.ApplicationModule.APPLICATION_MODULE_ID, moduleId);
-        ApplicationModule applicationModule = (ApplicationModule) applicationModuleRepository.select(QueryConstants.FIND_APPLICATION_MODULE, -1, 0, queryParameters);
+        ApplicationModule applicationModule = applicationModuleRepository.select(QueryConstants.FIND_APPLICATION_MODULE, -1, 0, queryParameters).get(0);
```

Comment:
Is it guaranteed this to return at least one value? Otherwise it is risky to directly call get(0)